### ISSUE TITLE
Drawing layer not supported on latest version of Core and Google

### ISSUE THREAD

**Issue by Cephyric-gh** [ng-maps/ng-maps#102]
The core and google libs list support for angular 15, however the drawing layer is still pointing to angular 8, which causes an NPM dependency error on install

```npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @ng-maps/drawing-layer@0.0.1
npm ERR! Found: @angular/common@15.0.4
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^15.0.4" from the root project
npm ERR!   peer @angular/common@"^15.0.0 || ^16.0.0" from @angular/cdk@15.0.4
npm ERR!   node_modules/@angular/cdk
npm ERR!     @angular/cdk@"^15.0.4" from the root project
npm ERR!     peer @angular/cdk@"15.0.4" from @angular/material@15.0.4
npm ERR!     node_modules/@angular/material
npm ERR!       @angular/material@"^15.0.4" from the root project
npm ERR!       1 more (@angular/material-moment-adapter)
npm ERR!     1 more (ng2-charts)
npm ERR!   12 more (@angular/forms, @angular/material, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^8.0.0" from @ng-maps/drawing-layer@0.0.1
npm ERR! node_modules/@ng-maps/drawing-layer
npm ERR!   @ng-maps/drawing-layer@"^0.0.1" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @angular/common@8.2.14
npm ERR! node_modules/@angular/common
npm ERR!   peer @angular/common@"^8.0.0" from @ng-maps/drawing-layer@0.0.1
npm ERR!   node_modules/@ng-maps/drawing-layer
npm ERR!     @ng-maps/drawing-layer@"^0.0.1" from the root project
```

Edit: Actually looks like the drawing layer isn't updated at all to support google's drawing manager? It references a part of core that doesn't exist anymore (GoogleMapsApiWrapper). Is there support for drawing manager in core, meaning the drawing layer isn't needed? This is currently a blocker preventing me from migrating from @agm-core to this, since it doesn't look like that is going to be updated to support the newer google SDK and angular 15 any time soon.

**Comment by DaSchTour** [comment]
Thanks for mentioning this. I'll take a look and create an new version of drawing-layer.
I focused on the core functionality till now.

**Comment by DaSchTour** [comment]
Published new version @ng-maps/drawing-layer@1.0.0 which has updates peer dependencies
